odoo/addons/base/models/ir_ui_view.py

```diff
@@ -1688,7 +1688,7 @@ def get_view_xmlid(self):
     @api.model
     def render_public_asset(self, template, values=None):
         template = self.sudo().browse(self.get_view_id(template))
-        template._check_view_access()
+        # template._check_view_access()
```

Comment:
Remove if not useful

Reply:
ho that's dangerous, it was something I removed with Lucas cause we were denied access to assets. Only for tests purposes, I need to add the line back.